### PR TITLE
CVE-2018-11760

### PR DESCRIPTION
**Pull request by squito**
ran `jekyll build` and `jekyll serve`, page looked OK locally.

### CODE REVIEW

File: security.md [hunk 1]
```diff
@@ -18,6 +18,36 @@ non-public list that will reach the Apache Security team, as well as the Spark P
...
+
+Versions affected:
+
+- All Spark 1.x, Spark 2.0.x, and Spark 2.1.x versions
+- Spark 2.2.0 to 2.2.2
+- Spark 2.3.0 to 2.3.1
+
+Description:
+
+When using PySpark , it's possible for a different local user
```
Comment (abellina) [inline security.md]: minor nit, extra space after PySpark

**Review by abellina — APPROVED** [review]
👍 had a minor formatting nit.

**Comment by squito** [comment]
oops, I didn't realize I should also commit the generated page, `site/security.html` as well, just updated that.  Oddly, `jekyll build` updated a few more pages under `site`, but I'm not including those changes here.

**Comment by abellina** [comment]
oh ok, sounds good. +1 non binding

**Comment by vanzin** [comment]
> jekyll build updated a few more pages under site

That's normal. There are indices and other stuff to update. Also maybe the person who made the previous change forgot to build the site, it happens sometimes.

Should probably add those back.

**Comment by squito** [comment]
so it turns out I just had an old version of jekyll installed.  After upgrading, then those other changes disappear.

**Comment by vanzin** [comment]
Cool, I'll merge this then.

**Comment by vanzin** [comment]
(Merge script seems broken, I'll just use the UI.)

**Comment by felixcheung** [comment]
sometimes it's the Latest News.

**Pull request merged**